### ISSUE TITLE
How is a single feature obtained from "bag of features" for tag prediction

### ISSUE THREAD

**Issue by sidhantls** [facebookresearch/StarSpace#310]
An entity is represented with a feature vector and a document is represented as a bag of features.  How is the bag of features aggregated to obtain the single document feature for training/evaluation? 

The loss function is calculated based on Sim(A, B) where A and B are vectors. Now, if A is a document and B is a single tag, how is the vector for A obtained? According to the paper, A is presented as a bag of features. Are the bag of features aggregated together with a mean operation to obtain A?
